docs/src/modules/utils/postProcessImport.test.ts

```diff
@@ -1,5 +1,7 @@
 import { expect } from 'chai';
-import { DATE_ADAPTER_VERSIONS, ADAPTER_TO_LIBRARY, postProcessImport } from './postProcessImport';
+// eslint-disable-next-line import/no-relative-packages
+import pickersPackageJson from '../../../../packages/x-date-pickers/package.json';
```

Comment:
Using `@mui/x-date-pickers/package.json` is not enough, because it doesn't have `devDependencies`. 🙈